### PR TITLE
Added conda recipe

### LINKED ISSUE #185: Add conda recipe
A conda recipe would be good to have.

### PR DESCRIPTION
**Pull request by sarthakpati**
Closes #185

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,6 +5,10 @@ BrotliCFFI
     :alt: Version
     :target: https://pypi.org/project/brotlicffi
 
+.. image:: https://anaconda.org/conda-forge/brotlicffi/badges/version.svg
```
Comment (sethmlarson) [inline README.rst:8]: I prefer the Shields SVG here: https://img.shields.io/conda/vn/conda-forge/brotlicffi

File: README.rst [hunk 2]
```diff
@@ -5,6 +5,10 @@ BrotliCFFI
...
+    :alt: Version-Conda
+    :target: https://anaconda.org/conda-forge/brotlicffi
+
```
Comment (sethmlarson) [inline README.rst:11]: Can we also add instructions on how to install from Conda Forge like so:

`$ conda install -c conda-forge brotlicffi`

**Comment by sarthakpati** [comment]
Better? 😄

**Review by sethmlarson — APPROVED** [review]
LGTM

**Pull request merged**